_sass/_objects--buttons.scss

```diff
@@ -11,3 +14,14 @@
     background-color: $color--btn-primary-hover;
   }
 }
+
+.btn-secondary {
+  padding: $size--secondary-padding-y $size--secondary-padding-x;
+
+  // TODO: Check if this is correct with design (Ask mari)
```

Comment:
I think so far you should know, and if it is a matter of asking let's wait for the answer before merge